tests/braille-specs/zhcn-cbs.yaml

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8 
+#
+# Chinese Common Braille Scheme
+#
+# according to Ministry of Education of the People's Republic of
+# China's website
+# <http://www.moe.gov.cn/jyb_sjzl/ziliao/A19/201807/W020180725666187054299.pdf>
+#
+# Copyright © 2022 by Cary-rowen <manchen_0528@outlook.com>
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# ----------------------------------------------------------------------------------------------
+
+display: unicode-without-blank.dis
+table:
+  language: cmn-Hans
+  region: cmn-CN
+  system: cmn-traditional
+  variant: chinese-common-braille-scheme
+  __assert-match: zhcn-cbs.ctb
+tests:
+  - [苹果,⠏⠡⠛⠕⠄]
+  - [电脑,⠙⠩⠝⠖⠄]
+  - [橘子,⠛⠬⠂⠵]
+  - [眼睛,⠩⠄⠛⠡⠁]
+  - [杯弓蛇影,⠃⠮⠁⠛⠲⠁⠱⠢⠂⠡⠄]
+  - [老马识途,⠇⠖⠄⠍⠔⠄⠱⠂⠞⠥]
+  - [相敬如宾,⠓⠭⠁⠛⠡⠚⠥⠃⠣⠁]
+  - [飞黄腾达,⠋⠮⠓⠶⠞⠼⠙⠔⠂]
+  - [如鱼得水,⠚⠥⠬⠂⠙⠢⠂⠱⠺⠄]
+  - [今天的天气不错,⠛⠣⠁⠞⠩⠁⠙ ⠞⠩⠁⠅⠊⠆⠃⠥⠉⠕⠆]
+  - [做人要有梦想,⠵⠕⠚⠴⠜⠳⠍⠼⠆⠓⠭⠄]
+  - [朝月亮出发,⠟⠖⠾⠇⠭⠟⠥⠁⠋⠔]
+  - [最深最重的爱，须与时日共成长。,⠵⠺⠱⠴⠁⠵⠺⠌⠲⠙ ⠪⠐ ⠓⠬⠁⠬⠄⠱⠂⠚⠆⠛⠲⠟⠼⠌⠦⠄⠐⠆]
+  - [这是我的国土,⠌⠢⠱⠕⠙ ⠛⠕⠂⠞⠥⠄]
+  - [hello,⠓⠑⠇⠇⠕]
+  - [Hello World,⠠⠓⠑⠇⠇⠕ ⠠⠺⠕⠗⠇⠙]
+  - [战略合作协议,⠌⠧⠇⠾⠓⠢⠵⠕⠓⠑⠂⠊⠆]
+  - [123,⠼⠁⠃⠉]
+  - [中国是发展中国家,⠌⠲⠁⠛⠕⠂⠱⠋⠔⠌⠧⠄⠌⠲⠁⠛⠕⠂⠛⠫⠁]
+  - [尊老爱幼是中华民族的传统美德,⠵⠒⠁⠇⠖⠄⠪⠳⠆⠱⠌⠲⠁⠓⠿⠍⠣⠵⠥⠂⠙ ⠟⠻⠞⠲⠄⠍⠮⠄⠙⠢⠂]
+
+# The differences with the current braille system is illustrated with the following examples:
+
+  # some braille cells can be omitted without causing ambiguity
+  # there are six shorthand words:
+
+  # 1. '他' ⠞⠔ is abbreviated ⠞
+  - [他们,⠞⠍⠴] # requires only 3 braille cells instead of 4
+  # 2. '它' ⠁⠞⠔ is abbreviated ⠁⠞
+  - [它们,⠈⠞⠍⠴] # requires only 4 braille cells instead of 6
+  # 3. '她' ⠞⠔⠁ is abbreviated ⠞⠁
+  - [她们,⠞⠁⠍⠴] # requires only 4 braille cells instead of 5
+  # 4. '你' ⠝⠊⠄ is abbreviated ⠝
+  - [你好,⠝⠓⠖⠄] # requires only 4 braille cells instead of 6
+  # ?
+  - [怎么,⠵⠴⠄⠍] # requires only 4 braille cells instead of 5
```

Comment:
@cary-rowen Is this not an example of a shorthand word?

Reply:
Yes,It is shorthand word

Reply:
OK, thanks for the correction.